spec/features/admin/admin_merchant_spec.rb

```diff
@@ -0,0 +1,13 @@
+require "rails_helper"
+
+RSpec.describe "Admin Merchant Page", type: :feature do
+  describe "When I visit the merchant index (/admin/merchants)" do
+    # US 24
+    it "I a list of all the merchants" do
+      merchant_1 = Merchant.create!(name: "Merchant 1")
+      visit admin_merchants_path
+
+      expect(page).to have_content("Merchant 1")
```

Comment:
Similarly, do we want to use `merchant_1.name` instead of hard-typing `Merchant 1`? I think using the variable names makes our tests stronger if ever change/add data.